tests/std/tests/P0122R7_span/test.cpp

```diff
@@ -591,58 +586,89 @@ constexpr bool test() {
 
     {
         static_assert(is_constructible_v<span<int>, ContiguousSizedRange&>);
-        static_assert(!is_constructible_v<span<int, 3>, ContiguousSizedRange&>);
+        static_assert(is_constructible_v<span<int, 3>, ContiguousSizedRange&>);
 
         static_assert(!is_constructible_v<span<int>, NonRange&>);
+        static_assert(!is_constructible_v<span<int, 3>, NonRange&>);
 
         static_assert(!is_constructible_v<span<int>, const ContiguousSizedRange&>);
+        static_assert(!is_constructible_v<span<int, 3>, const ContiguousSizedRange&>);
         static_assert(is_constructible_v<span<const int>, ContiguousSizedRange&>);
+        static_assert(is_constructible_v<span<const int, 3>, ContiguousSizedRange&>);
         static_assert(is_constructible_v<span<const int>, const ContiguousSizedRange&>);
+        static_assert(is_constructible_v<span<const int, 3>, const ContiguousSizedRange&>);
         static_assert(!is_constructible_v<span<double>, ContiguousSizedRange&>);
+        static_assert(!is_constructible_v<span<double, 3>, ContiguousSizedRange&>);
 
         static_assert(is_convertible_v<ContiguousSizedRange&, span<int>>);
+        static_assert(is_convertible_v<ContiguousSizedRange&, span<int, 3>>);
         static_assert(is_convertible_v<ContiguousSizedRange&, span<const int>>);
+        static_assert(is_convertible_v<ContiguousSizedRange&, span<const int, 3>>);
         static_assert(is_convertible_v<const ContiguousSizedRange&, span<const int>>);
+        static_assert(is_convertible_v<const ContiguousSizedRange&, span<const int, 3>>);

```

Comment:
These newly added `is_convertible_v<RANGE, static extent span>` tests look like they shouldn't be compiling, due to the conditional `explicit` on the range constructor. Are they actually passing?

Reply:
Yes, thes should be inverted as the explicit prevents convertability